### PR TITLE
kdevelop: Init at 5.0

### PR DESCRIPTION
**Pull request by ambrop72**
###### Motivation for this change

New major release.
###### Things done
- [x] Tested using sandboxing
  ([nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS,
    or option `build-use-chroot` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Doesn't use Okteta yet.

### CODE REVIEW

**Comment by FRidh** [comment]
Could you add license information.

**Comment by ambrop72** [comment]
Added licences.

File: pkgs/applications/editors/kdevelop5/kdevelop.nix [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{ stdenv, fetchurl, cmake, gettext, pkgconfig, extra-cmake-modules, makeQtWrapper
+, qtquickcontrols, qtwebkit
+, kconfig, kdeclarative, kdoctools, kiconthemes, ki18n, kitemmodels, kitemviews
+, kjobwidgets, kcmutils, kio, knewstuff, knotifyconfig, kparts, ktexteditor
+, threadweaver, kxmlgui, kwindowsystem
+, plasma-framework, krunner, kdevplatform, kdevelop-pg-qt, shared_mime_info
+, libksysguard, /* okteta, */ llvmPackages
```
Comment (grahamc) [inline pkgs/applications/editors/kdevelop5/kdevelop.nix]: Let's remove this comment prior to merging.

**Comment by ambrop72** [comment]
Removed comments about okteta.

**Comment by grahamc** [comment]
:lgtm:

---

Reviewed 4 of 4 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/nixos/nixpkgs/18031#-:-KQGCiCNWjadpOkvnCBA:bnfp4nl)*

**Pull request merged**